package.json

```diff
@@ -35,6 +35,7 @@
     "react-feather": "^2.0.10",
     "react-redux": "^8.1.2",
     "react-toastify": "^9.1.3",
+    "sharp": "^0.32.6",
```

Comment:
package installed but not used anywhere. Can you remove this if you are not using?